### PR TITLE
Bug fix: #4011 #4012 - "show run acl", "show run interfaces" - traceback when no ACL_RULE/INTERFACE is present.

### PR DESCRIPTION
**Pull request by noaOrMlnx**
Signed-off-by: Noa Or <noaor@mellanox.com>

**- What I did**
- Fix bug #4011 - "show run acl" returns traceback when no ACL_RULE is present in configuration.
- Fix bug #4012 - "show run interfaces" returns traceback when no INTERFACE is present in configuration

**- How I did it**
Change logic in main to take data only if the var given in var_json appears in DB.

**- How to verify it**
- Make sure running configuration doesn't contain ACL_RULE:
`show run all | grep -r '"ACL_RULE"'`
should print nothing.
- run command:
`show run acl`
Nothing should be printed and no error should appear.

- Make sure running configuration doesn't contain INTERFACE:
`show run all | grep -r '"INTERFACE"'`
should print nothing.
- run command:
`show run interfaces`
Nothing should be printed and no error should appear.

**- Description for the changelog**

N/A

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: src/sonic-config-engine/sonic-cfggen [hunk 1]
```diff
@@ -298,7 +298,7 @@ def main():
     if args.var_json != None:
         if args.key != None:
             print(json.dumps(FormatConverter.to_serialized(data[args.var_json], args.key), indent=4, cls=minigraph_encoder))
-        else:
+        elif args.var_json in data:
```
Comment (jleveque) [inline src/sonic-config-engine/sonic-cfggen]: The `if` statement also references `data[args.var_json]`. Maybe this check should be done first, e.g.,

```
    if args.var_json != None and args.var_json in data:
        if args.key != None:
            print(json.dumps(FormatConverter.to_serialized(data[args.var_json], args.key), indent=4, cls=minigraph_encoder))
        else:
            print(json.dumps(FormatConverter.to_serialized(data[args.var_json]), indent=4, cls=minigraph_encoder))
```

Comment (noaOrMlnx) [reply src/sonic-config-engine/sonic-cfggen]: Done.

**Comment by noaOrMlnx** [comment]
retest vsimage please

**Comment by noaOrMlnx** [comment]
retest vsimage please

**Comment by noaOrMlnx** [comment]
retest vsimage please

**Comment by liat-grozovik** [comment]
@jleveque can you please review unittest and approve?

**Pull request merged**